File: college/day-department/algorithms/2 sorting_algorithms/3_selection_sort/selection_sort.cs
using System;

class Program
{
    static void SelectionSort(int[] arr)
    {
        int n = arr.Length;
        for (int i = 0; i < n - 1; i++)
        {
            int min_index = i;
            for (int j = i + 1; j < n; j++)
            {
                if (arr[j] < arr[min_index])
                {
                    min_index = j;
                }
            }
            int temp = arr[min_index];
            arr[min_index] = arr[i];
            arr[i] = temp;
        }
    }

    static void Main(string[] args)
    {
        int[] arr = { 64, 25, 12, 22, 11 };
        SelectionSort(arr);
        Console.Write("Sorted array is: ");
        foreach (int num in arr)
        {
            Console.Write(num + " ");
        }
    }
}

File: 2 sorting_algorithms/3_selection_sort/selection_sort.py
def selection_sort(arr):
    n = len(arr)
    for i in range(n):
        min_index = i
        for j in range(i+1, n):
            if arr[j] < arr[min_index]:
                min_index = j
        arr[i], arr[min_index] = arr[min_index], arr[i]

# Example usage:
arr = [13, 2, 64, 25, 12, 22, 11]
selection_sort(arr)
print("Sorted array is:", arr)
